expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionService.java

```diff
@@ -12,4 +13,6 @@
     ExpandedDataEntry expandEntry(DataEntry dataEntry) throws JsonProcessingException, NotFoundException;
 
     Set<URI> getOrganizationIds(DataEntry dataEntry) throws NotFoundException;
+
+    URI getResourceId(SortableIdentifier identifier);
```

Comment:
I do not think that this method is necessary. Then we create some weird dependencies where the data entry needs the service and the service needs the data entries.
Suggestion:
```
public static ExpandedDoiRequest create(DoiRequest doiRequest,
                                            ResourceExpansionService resourceExpansionService)
        throws NotFoundException {
        ExpandedDoiRequest expandedDoiRequest = ExpandedDoiRequest.fromDoiRequest(doiRequest);
        Set<URI> ids = resourceExpansionService.getOrganizationIds(doiRequest);
        expandedDoiRequest.setOrganizationIds(ids);
        URI resourceId =  new UriWrapper(ExpansionConstants.ID_NAMESPACE)
            .addChild(expandedDoiRequest.getIdentifier().toString()).getUri();
        expandedDoiRequest.setResourceId(resourceId);
        return expandedDoiRequest;
    }
```